<launch>
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find scout_service)/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find scout_service)/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find scout_service)/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find scout_service)/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find scout_service)/service_local_planner_params.yaml" command="load" />

    <remap from="map" to="/map" />
    
  	<!-- START: Use with estop -->
<!--	  <remap from="cmd_vel" to="cmd_vel_estop"/>-->
    <!-- END: Use with estop -->
  </node>
</launch>
